sdk/src/types/block/output/rent.rs

```diff
@@ -31,145 +36,153 @@ const DEFAULT_BYTE_COST_FACTOR_BLOCK_ISSUER_KEY: u8 = 1;
     serde(rename_all = "camelCase")
 )]
 pub struct RentStructure {
```

Comment:
This has been renamed to RentParameters.
https://github.com/iotaledger/tips-draft/pull/25/files#diff-bf3729cb12f375685fc23f726a6be49461ce3df320037f6639b323dabfebf651R476
https://github.com/iotaledger/iota.go/blob/serix-iota-core/rent.go#L36